frontend/openchat-agent/src/services/user/mappers.ts

```diff
@@ -761,7 +761,7 @@ function mention(candid: ApiMention): Mention {
     };
 }
 
-function groupChatSummary(candid: ApiGroupChatSummary, limitReadByMeUpTo = true): GroupChatSummary {
```

Comment:
`limitReadByMeUpTo` was only needed temporarily when we had a bug which had marked some `readUpTo` values way too high